src/ubuntu/16.04/crossdeps/Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-coredeps
 
 # Install the base toolchain we need to build anything (clang, cmake, make and the like).
 RUN apt-get update \
+    wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc | apt-key add - && \
```

Comment:
This syntax doesn't look right,  I was expecting a `&&` after the initial `apt-get update`.  Is the initial call to `apt-get update` even necessary?  Please put the `&&` at the beginning of the lines as this is the convention typically used within the Dockerfiles of this repo.